FLT/ForMathlib/MeasurableSpacePadics.lean

```diff
@@ -0,0 +1,6 @@
+import Mathlib
+
+variable {p : ℕ} [Fact p.Prime]
+
+instance : MeasurableSpace ℚ_[p] := sorry
+instance : BorelSpace ℚ_[p] := sorry
```

Comment:
I think that this might be over-reach for this PR. Can we just get the character I need in?

Reply:
What is the character you need? Two weeks you said we should compute this canonical norm for the reals, complexes, p-adic numbers and integers, which is what Javier and I tried doing

Reply:
I think this PR should just stick to getting `DistribHaarCharacter` over the line. As you say, I need the p-adics, reals and complexes, but those can come later.